bigbluebutton-html5/imports/api/acl/Acl.js

```diff
@@ -26,6 +26,10 @@ export default class Acl {
       meetingId,
       userId,
       authToken,
+      validated: true,
+      // TODO: We cant check for approved until we move subscription logif out of <Base />
```

Comment:
The approved logic is done in `<App/>` thus we can`t check for approved, otherwise it will trigger a throw on the `<Base/>` since the user can`t subscribe.